templates/cinder/config/cinder.conf

```diff
@@ -79,3 +79,12 @@ lock_path = /var/lib/cinder/tmp
 #volume_backend_name=nfs
 #volume_driver=cinder.volume.drivers.nfs.NfsDriver
 #nfs_shares_config=/etc/cinder/nfs_shares
+
+[ceph]
+volume_backend_name=ceph
+volume_driver=cinder.volume.drivers.rbd.RBDDriver
+rbd_ceph_conf=/etc/ceph/ceph.conf
+rbd_user=openstack
+rbd_pool=volumes
+rbd_flatten_volume_from_snapshot=False
+rbd_secret_uuid={{ .CephClusterFSID }}
```

Comment:
Please forgive my ignorance here, but is there anything else in this section that we might want to parameterize in addition to `rbd_secret_uuid`?  I'm not familiar with Ceph to any significant extent, but I'm curious if `rbd_user`, `rbd_pool`, etc, could vary between different external Ceph deployments.

Reply:
Yes, most definitely. We'll be submitting follow up PRs to parameterize it all. Same for glance.

So do we do that in one PR or in a few follow ups?

Reply:
> Please forgive my ignorance here, but is there anything else in this section that we might want to parameterize in addition to `rbd_secret_uuid`? I'm not familiar with Ceph to any significant extent, but I'm curious if `rbd_user`, `rbd_pool`, etc, could vary between different external Ceph deployments.

Yeah, I'm updating this PR (since it's already here) to add the more parameters. @fultonj feel free to make the same thing for Glance.